src/command_modules/azure-cli-documentdb/tests/test_documentdb_commands.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, JMESPathCheck, ResourceGroupPreparer
+
+
+class DocumentDBTests(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_create_database_account(self, resource_group):
+        name = self.create_random_name(prefix='cli', length=40)
+        self.cmd(
+            'az documentdb create -n {} -g {} '
+            '--enable-automatic-failover {} --default-consistency-level {}'
+            .format(name, resource_group, 'true', 'ConsistentPrefix'))
+        self.cmd('az documentdb show -n {} -g {}'.format(name, resource_group), checks=[
+            JMESPathCheck('properties.enableAutomaticFailover', 'true'),
+            JMESPathCheck('properties.consistencyPolicy.defaultConsistencyLevel',
+                          'ConsistentPrefix'),
+        ])
+        self.cmd('az documentdb list -g {}'.format(resource_group))
+        self.cmd('az documentdb list-keys -n {} -g {}'.format(name, resource_group))
+        self.cmd('az documentdb regenerate-key -n {} -g {} --key-kind {}'
+                 .format(name, resource_group, 'primary'))
+        self.cmd('az documentdb delete -n {} -g {}'.format(name, resource_group))
```

Comment:
Add proper verification to above four commands. Testing exit code only is not enough.